docker/nginx/README.md

```diff
@@ -0,0 +1,23 @@
+## Docker image that compiles NGINX from source
+
+The nginx Headers More module doesn't currently accept wildcard matches
+for clearing input headers (it's on a branch somehwere). So if we want to use this
```

Comment:
it's on a branch somewhere????? either link to it or don't include the parens. (also somewhere typo)